chainweb.cabal

```diff
@@ -397,7 +397,8 @@ library
         , filepath >= 1.4
         , ghc-compact >= 0.1
         , growable-vector >= 0.1
-        , hashable >= 1.4
+        -- hash of bytestrings changed
+        , hashable >= 1.4 && < 1.4.5
```

Comment:
We used to track upper bounds in cabal.project, where it is easier to group them and keep them in a single place together.